INTRODUCTION.md

```diff
@@ -723,65 +723,27 @@ This method should be called by the application on delivery report error.
 
 Using the transactional producer simplifies error handling compared to the
 standard or idempotent producer, a transactional application will only need
-to care about two different types of errors:
+to care about these different types of errors:
 
- * Fatal errors - the application must cease operations and destroy the
-   producer instance if any of the transactional APIs return
-   `RD_KAFKA_RESP_ERR__FATAL`. This is an unrecoverable type of error.
+ * Retriable errors - the operation failed due to temporary problems,
+   such as network timeouts, the operation may be safely retried.
+   Use `rd_kafka_error_is_retriable()` to distinguish this case.
  * Abortable errors - if any of the transactional APIs return a non-fatal
    error code the current transaction has failed and the application
    must call `rd_kafka_abort_transaction()`, rewind its input to the
    point before the current transaction started, and attempt a new transaction
    by calling `rd_kafka_begin_transaction()`, etc.
+   Use `rd_kafka_error_is_txn_abortable()` to distinguish this case.
+ * Fatal errors - the application must cease operations and destroy the
+   producer instance.
+   Use `rd_kafka_error_is_fatal()` to distinguish this case.
+ * For all other errors returned from the transactional API: the current
+   recommendation is to treat any error that has neither retriable, abortable,
+   or fatal set, as a fatal error.
```

Comment:
this all reads well to me. fwiw, what i'm wondering about having read this is what to do with errors that occurred whilst using the transactional producer from methods that aren't part of the transactional api.